content/authors/frank-joseph/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Frank Joseph
+type: authors
+twitter: https://twitter.com/josepfrank05/
+---
+Frank Joseph is an innovative software developer building stuffs that run on the internet. I write React, JavaScript, Java and Spring boot.Techpreneur at Semicolon Africa.
```

Comment:
I don't like the word *stuffs* here, it sounds vague and unprofessional. I would change it to *applications* or *apps*. Also, you are missing a space here:

>boot.Techpreneur

after the period.

Reply:
Done